src/api/app/assets/stylesheets/webui/application/kiwi_image.scss

```diff
@@ -0,0 +1,56 @@
+#update_form {
```

Comment:
@hennevogel 

> This would be a bit too generic for my taste. And you do not style the form, only .repository-container and children.

Changed `update_form` by `kiwi_repository_update_form`, and changed `.repository-container` by `.kiwi-repository-container`. Better?